### DISCUSSION TITLE
Hekili Add-On Resets on Reload

### DISCUSSION THREAD

**Discussion by blecaplol** [SFX-WoW/Masque#226] [Issues] [answered]
### Game Flavor

Retail

### Game Version

9.1.5

### Add-On Version

Latest

### Describe the Issue

An external addon call Hekili is not always skinned

### Steps To Reproduce

You can just load it, select the skin, for me Dream and then reload again. It went back to default now.

**Comment by StormFX** [comment]
This is likely a write issue with your save variables for Masque. Try deleting them.

**Comment by blecaplol** [comment]
> This is likely a write issue with your save variables for Masque. Try deleting them.

I just did and no joy. Still went back default after reloading.

ps However, I click enable twice on the addon on the minimap not a /reload and it got skinned but it is not persistent to me.

**Comment by StormFX** [comment, marked as answer]
Ok. Try selecting the "Global" section of the skinning tab in Masque and clicking the reset button. Then, select a different skin, reload and see if it sticks. Then you can re-select your preferred skin. If that doesn't work, it may be an issue with how Hekii is implementing it. It works with everything else on my end, so it's not Masque issue.

**Comment by blecaplol** [comment]
> Ok. Try selecting the "Global" section of the skinning tab in Masque and clicking the reset button. Then, select a different skin, reload and see if it sticks. Then you can re-select your preferred skin. If that doesn't work, it may be an issue with how Hekii is implementing it. It works with everything else on my end, so it's not Masque issue.

Ok I did ask instructed. Reset, select a different skin "Zoomed" followed by /reload it i still the same. Again, I immediately click Enable twice on Hekili and the skin came back differently. This has never happened before but until an update somewhere in between, i'm just saying what I noticed. It's ok thx for help.
